sdk/core/core-amqp/src/retry.ts

```diff
@@ -111,38 +111,39 @@ async function checkNetworkConnection(host: string): Promise<boolean> {
 }
 
 /**
- * It will attempt to linearly retry an operation specified number of times with a specified
- * delay in between each retry. The retries will only happen if the error is retryable.
+ * Additional attempts are made if the previous attempt failed with a `retryable` error.
```

Comment:
Your previous version had a line before this line talking about the initial attempt. That would set good context to talk about "addition attempts" in the next line. Any reason to remove it?

Something like `Every operation is attempted atleast once` right before talking about `Additional attempts` would be nice

Reply:
Also the word retryable doesn't need the back quotes here